_data/hints.yml

```diff
@@ -29,7 +29,7 @@ image_asset:
   logo_alt_text: "If you leave this blank, we'll use the site name provided on the Settings page as the default alt text."
 
 watcher:
-  name: "Enter a short, meaningful name - this will appear in the subject line of your alert."
+  name: "Enter a short, meaningful name; this will appear in the subject line of your alert."
```

Comment:
@dawnpm Where does this appear on the site?

Reply:
These hints are the tooltip flags in the admin center. They get piped in.